armi/reactor/blocks.py

```diff
@@ -2167,6 +2175,25 @@ def _rotatePins(self, rotNum, justCompute=False):
 
         return rotateIndexLookup
 
+    def _rotatePinParameters(self, rotNum: int):
```

Comment:
Could use a one-line comment. WE had to talk about this a lot, so there must be something interesting to say about it.

Up to you though; not a deal breaker.

Reply:
Good with me

Reply:
`HexBlock.rotate` docs heavily updated in https://github.com/terrapower/armi/pull/1877/commits/28d1a353f425f8f9e6b32c42d69e5d19afade001

This specific method doc'ed in 2ddc312de9eacafed79224af42c5ef3651e0e115